testcases/commands/ar/ar01

```diff
@@ -264,14 +264,14 @@ test13()
 test14()
 {
 	ROD touch file0.in
-	ROD ar -cr lib.a file0.in $TST_DATAROOT/file1.in
+	ROD ar -crU lib.a file0.in $TST_DATAROOT/file1.in
```

Comment:
the `U` flag is not in POSIX which means it's only supported by some toolchains and is not portable.  so you're actually making things worse for embedded systems ;).

http://pubs.opengroup.org/onlinepubs/9699919799/utilities/ar.html